rdt/transformers/pii/anonymizer.py

```diff
@@ -85,7 +85,7 @@ def _check_locales(self):
             )
 
     def __init__(self, provider_name=None, function_name=None, function_kwargs=None,
-                 locales=None, enforce_uniqueness=False):
+                 locales=None, enforce_uniqueness=False, missing_value_generation='random'):
```

Comment:
can we add this to the docstring as well?

Reply:
Yes, done in [b597b31](https://github.com/sdv-dev/RDT/pull/669/commits/b597b315a4fd720bbb8a8a9ce6dd58191112b50d)